core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -413,6 +413,34 @@ package object config {
       .intConf
       .createWithDefault(1)
 
+  private[spark] val STORAGE_DECOMMISSION_ENABLED =
+    ConfigBuilder("spark.storage.decommission.enabled")
+      .doc("Whether to decommission the block manager when decommissioning executor")
+      .version("3.1.0")
+      .booleanConf
+      .createWithDefault(false)
+
+  private[spark] val STORAGE_DECOMMISSION_MAX_REPLICATION_FAILURE_PER_BLOCK =
+    ConfigBuilder("spark.storage.decommission.maxReplicationFailuresPerBlock")
```

Comment:
I think we can just use `spark.storage.maxReplicationFailures` directly. Less configurations contribute to better UX.

Reply:
So I'm not sure that's a great idea. Looking at `maxReplicationFailures` the default is set to one, which certainly makes sense in the situation where we don't expect the host to be exiting. But this situation is different, we know the current block is going to disappear soon so it makes sense to more aggressively try and copy the block.

Reply:
I see, thanks for your explanation.

>  But this situation is different, we know the current block is going to disappear soon so it makes sense to more aggressively try and copy the block.

Reply:
I see, thanks for your explanation.